bedrock/base/templates/macros.html

```diff
@@ -474,3 +474,31 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
     </footer>
   </section>
 {%- endmacro %}
+
+
+{% macro google_custom_search(search_type, type_class, default_term, term_list) %}
+{#
+  Parameters -
+    search_type:  The 'cx' hash for the specific custom search configuration; get this from cse.google.com.
+                  Defaults to general search.
+
+    search_class: A class added to the bar for use as a style hook to customize the presentation.
+
+    term_list:    A comma separated list of suggested terms displayed in a loop via JS.
+
+    default_term: A static suggested term to avoid an empty label if JS fails. This should
+                  usually be the same as the last term in the list (last to avoid clumsy repetition
+                  when JS kicks in to start the cycle).
+
+  Search engines -
+    General: 014783244707853607354:ifn1iyzxd1a
```

Comment:
Seems like a good opportunity to have these IDs in settings. Would be a bit more of a pain to go around the code changing them in every use location instead of just in settings. I'm thinking you could have something like this in settings:

```
GOOG_CUSTOM_SEARCH_TYPES = {
    'general': '014783244707853607354:ifn1iyzxd1a',
    'developer': '014783244707853607354:s-9zxgjd2o0',
    'activist': '014783244707853607354:bbpl3emdsii',
}
```

Then just have each call to the macro pass in "general", "developer", or "activist" and get the real value out of the setting: `{{ settings.GOOG_CUSTOM_SEARCH_TYPES[search_type] }}`. That would clean up the calls to the macro quite a bit and make it more obvious just looking at the template which one is being used.